docs/Technical Reference.md

```diff
@@ -2,110 +2,102 @@ This page provides a technical reference on certain advanced configuration optio
+    </CORSRule>
+    </CORSConfiguration>
+    ```
 
-      ```xml
-      <?xml version="1.0" encoding="UTF-8"?>
-      <CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
-      <CORSRule>
-          <AllowedOrigin>https://yourcompanydomain.com</AllowedOrigin>
-          <AllowedMethod>GET</AllowedMethod>
-          <AllowedMethod>HEAD</AllowedMethod>
-          <AllowedMethod>PUT</AllowedMethod>
-          <AllowedMethod>POST</AllowedMethod>
-          <AllowedHeader>*</AllowedHeader>
-          <MaxAgeSeconds>3000</MaxAgeSeconds>
-      </CORSRule>
-      </CORSConfiguration>
-      ```
+    > Note: bucket CORS does not grant permissions of any kind.
+    > `AllowedMethod` actions are only available to IAM users or roles with sufficient permissions.
 
-      The above policy makes it possible for your users to browse buckets on the web.
-      > Note: bucket CORS does not grant permissions of any kind.
-      > `AllowedMethod`s are only callable by IAM users or roles with sufficient permissions.
-1. *ARN for server certificate* - We require HTTPS for access to the Quilt web catalog. You may use the CloudFront domain output by CloudFormation. Alternatively, if you wish to use a custom domain, create or upload an [SSL/TLS server certificate](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_server-certs.html).
-1. *ARNs for search users* - Only select users that you specifcy can search S3. Search users can view contents for any bucket object, so only add users that you trust to this list.
+    Note the `AllowedOrigin` field. This should be parameterized with the domain you will host your catalog from.
 
-### Permissions
+    If you are going to use Quilt T4 with a new bucket, create the bucket now, and set these policies as part of the flow for bucket creation.
 
-Your users use their IAM credentials to access T4 on the web and in Python.
+3. Create, or ensure you have already created, an [AWS Certificate](https://aws.amazon.com/certificate-manager/) which maps to the public domain name you want your catalog to use. For example, if you want your catalog to be publicly accessible from `t4.foo.com`, you will need to have a certificate for `t4.foo.com` or `*.foo.com` registered in your account.
 
-Your designated Search users will be able *to search and preview file contents
-on anything in your T4 bucket*. **Only enable search for users who should be
-able to see the entire bucket contents**.
+   An AWS certificate is just an Amazon-issued HTTPS certificate, and it's a necessity because it enables HTTPS access to your catalog. If you have not created one, [step through the flow for creating one now](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html). If you already have a certificate for your website, but it's not an AWS-issued certificate, see the instructions on [importing an external certificate into AWS](https://docs.aws.amazon.com/acm/latest/userguide/import-certificate.html).
```

Comment:
AWS Certificate is ambiguous and if you Google that it will tell you how to get AWS training.

We should mention and link to AWS Certificate Manager service (https://aws.amazon.com/certificate-manager/) and state that we are specifically looking for a TLS certificate.